glide-core/src/client/mod.rs

```diff
@@ -397,9 +405,16 @@ fn sanitized_request_string(request: &ConnectionRequest) -> String {
     let client_name = chars_to_string_option(&request.client_name)
         .map(|client_name| format!("\nClient name: {client_name}"))
         .unwrap_or_default();
-
+    let periodic_checks_interval = if request.periodic_checks_interval != 0 {
+        format!(
+            "\nperiodic_checks_interval: {:?}",
```

Comment:
this will log `periodic_checks_interval: -1`, instead of a more informative: `periodic_checks_interval: disabled`.